google/cloud/pubsub/internal/tracing_message_batch.cc

```diff
@@ -110,7 +110,7 @@ Spans MakeBatchSinkSpans(Spans message_spans) {
   batch_sink_spans.push_back(MakeParent({{}}, message_spans));
   auto batch_sink_parent = batch_sink_spans.front();
 
-  auto cut = [&message_spans](auto i) {
+  auto cut = [&message_spans, &kMaxOtelLinks](auto i) {
```

Comment:
An alternative might be to promote `kMaxOtelLinks` to a namespace-scope constexpr, rather than having it buried away as a local, and despite it being used only once.

Reply:
Eventually the value will be set via options, so I think it's best to make it not a constexpr for now.

Reply:
Will it also be an `int64_t` then?  It seems like every time we use it we need to cast it to another type, so it might be nice to choose the best type now.